etc/desi.json

```diff
@@ -6,6 +6,23 @@
     },
     "datachallenge":{
         "exclude":[],
+        "dc17a-twopct":{
+            "dc17a-twopct/[README|twopct.ecsv|dc17a-lite.tar.gz]":"dc17a-twopct/toplevel.tar",
+            "dc17a-twopct/batch/.*":"dc17a-twopct/batch.tar",
+            "dc17a-twopct/fiberassign/.*":"dc17a-twopct/fiberassign.tar",
+            "dc17a-twopct/spectro/redux/dc17a2/exposures/201908.*":"dc17a-twopct/spectro/redux/dc17a2/exposures/201908nn.tar",
```

Comment:
Why don't you just back up the individual exposure directories?  That would be much easier.
```json
"dc17a-twopct/spectro/redux/dc17a2/exposures/([0-9]{8})/.*$": "dc17a-twopct/spectro/redux/dc17a2/exposures/\\1.tar",
```